README.md

```diff
@@ -239,7 +239,7 @@ class MyAction(Builder.Action):
 
 This can be run with ```builder.pyz my-action``` or ```builder.pyz myaction``` or ```builder.pyz MyAction```
 
-See api.py for the available API to actions. See https://github.com/awslabs/aws-crt-python/tree/master/.builder/actions for examples.
+See api.py for the available API to actions. See https://github.com/awslabs/aws-crt-python/tree/main/.builder/actions for examples.
```

Comment:
`https://github.com/awslabs/aws-crt-python/tree/main/.builder/actions` this link is not existed? Should we stay with master for this?

Reply:
I'm going to update the repo. I know this is a little premature, but it won't be in a couple hours.